packages/python-vtapi3/PKGBUILD

```diff
@@ -0,0 +1,39 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-vtapi3
+_pkgname=virustotalapi3
+pkgver=259.7d9e278
+pkgrel=1
+pkgdesc='The module that implements the VirusTotal API functions (3 version).'
+arch=('any')
+url='https://github.com/drobotun/virustotalapi3'
+license=('MIT')
+depends=('python' 'python-requests')
+makedepends=('python-setuptools')
```

Comment:
Can you test tour packages before submitting a PR please?

```
$ ba-dev -b
Syncing chroot copy /home/noraj/blackarch_chroot/blackarch/ with /home/noraj/blackarch_chroot/root/ ...
[sudo] password for noraj: 
Chroot environment is ready!
Consideer installing pkgcheck with pip install pkgcheck-arch --user for automatic PKGBUILD syntax error checking.
==> Making package: python-vtapi3 259.7d9e278-1 (sam. 14 janv. 2023 15:09:13)
==> Retrieving sources...
  -> Cloning virustotalapi3 git repo...
Cloning into bare repository '/home/noraj/Dev/blackarch/packages/python-vtapi3/virustotalapi3'...
remote: Enumerating objects: 975, done.
remote: Counting objects: 100% (105/105), done.
remote: Compressing objects: 100% (70/70), done.
remote: Total 975 (delta 61), reused 60 (delta 35), pack-reused 870
Receiving objects: 100% (975/975), 215.31 KiB | 2.31 MiB/s, done.
Resolving deltas: 100% (611/611), done.
==> Validating source files with sha512sums...
    virustotalapi3 ... Skipped
==> ERROR: Cannot find the git package needed to handle git sources.
==> ERROR: Build failed, check /home/noraj/blackarch_chroot/blackarch/build
Failed to build the package.
```

For example here, you missed the `git` make dependency.